aten/src/ATen/cuda/CUDASparseBlas.cpp

```diff
@@ -392,6 +392,78 @@ void bsrmv<c10::complex<double>>(
       reinterpret_cast<cuDoubleComplex*>(y)));
 }
 
+template <>
+void linear_solve<float>(
+  CUSOLVER_LINEAR_SOLVE_ARGTYPES(float)) {
+    TORCH_CUSOLVER_CHECK(cusolverSpScsrlsvluHost(
+        handle,
+        n,
+        nnzA,
+        descrA,
+        reinterpret_cast<const float*>(csrValA),
```

Comment:
This `reinterpret_cast` does nothing because `csrValA` is already of type `const float*`.

Reply:
Yup, removed now. Thanks! :)